ocs_ci/framework/conf/default_config.yaml

```diff
@@ -201,6 +201,8 @@ ENV_DATA:
   opm_owner_repo: "operator-framework/operator-registry"
   # opm tool release tag (particular version or "latest") to be downloaded
   opm_release_tag: "latest"
+  # virtctl tool github owner and repository (used in api url for downloading opm tool)
+  virtctl_owner_repo: "kubevirt/kubevirt"
```

Comment:
Do you think it is a good idea to download the virtctl binary from the existing cluster rather than from the upstream repository?
I have implemented something similar, where it retrieves the virtctl binary download links from the existing cluster, gets the virtctl download URL based on the specified platform and architecture, and then downloads and extracts the virtctl binary to bin_dir. This process is part of the CNV deployment [here](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/cnv.py#L631)
If there is a specific need to download the virtctl binary ONLY from the upstream repository, please ignore this comment.

Reply:
@yitzhak12 WDYT about this approach?

Reply:
Yes, it should be fine. We need to confirm this code will also be executed in a multicluster job test.

Reply:
Updated the logic, that it firstly try to get it from the CNV if deployed on the cluster and if that fails, it downloads it from upstream github repository.